### ISSUE TITLE
Wither doesn't damage blocks on regions

### ISSUE THREAD

**Issue by LaoTheLizard** [EngineHub/WorldGuard#1671]
**Versions**

WorldEdit version:
7.2.0
WorldGuard version:
7.0.4
Platform version:
Paper 288 (1.16.4)

**Describe the bug**

Wither doesn't destroy obsidian blocks when he follow me in region where I'm not a member/owner.
If I turn /op on - he destroy blocks by witherheads and time to time destroy obsidian blocks by its corpse. If I turn /op off - he doesn't destroy any blocks in region, but continues to destroy beyond.

**To Reproduce**

1. make a region for another player
2. go in that region (you are not an operator, you're not a member/owner)
3. create wither somewhere nearby
4. damage him to 1/2 or less HP
5. target him to you while you're walking in this region

**Expected behavior**

As you'll see later, I tuned all to make able to destroy all blocks in regions by wither. But I can't reach it. What should I do?

**Screenshots**

I set my config.yml like that: https://prnt.sc/vphc62
I tuned global region like here: https://prnt.sc/vphb1t
And this is the region where I tested it after: https://prnt.sc/vphbe9
(player SteelNat is not me)
This is a scene (no obsidian, no other blocks is destroyed): https://prnt.sc/vphgb3
And this is a same scene, when that region was just deleted (obsidian damaged): https://prnt.sc/vphhho and https://prnt.sc/vphhvt

**Additional context**

**Comment by wizjany** [comment]
...yes? I don't understand what the issue is? do you just want to unprotect the region entirely?

**Comment by LaoTheLizard** [comment]
> ...yes? I don't understand what the issue is? do you just want to unprotect the region entirely?

Yes, I want to unprotect the region from wither. By the way, WG has flags and settings to make it able. But it doesn't work.

**Comment by wizjany** [comment]
what you're asking is to unprotect the region from any player, as the player is counted as having caused the damage since they led the wither to break blocks in the region.
if worldguard didn't work like this, anyone could blow up tnt in other people's regions because they technically didn't break the block - the tnt did.

**Comment by LaoTheLizard** [comment]
> what you're asking is to unprotect the region from any player, as the player is counted as having caused the damage since they led the wither to break blocks in the region.
> if worldguard didn't work like this, anyone could blow up tnt in other people's regions because they technically didn't break the block - the tnt did.

I want the next:
1. Other players can't break blocks.
2. Other players can't use doors, chest, etc.
3. Other players can grief region through "TNT-cannon".
4. Other players can grief region by wither targeting.
5. Other players can grief region by pistons.

And I'm following to that by the next:
CONFIG.YML:
block-creeper-explosions: false
    block-creeper-block-damage: false
    block-wither-explosions: false
    block-wither-block-damage: false
    block-wither-skull-explosions: false
    block-wither-skull-block-damage: false
    block-other-explosions: false
    block-tnt: false
    block-tnt-block-damage: false

And after that I added flags to global region like here: https://prnt.sc/vphb1t
(pistons allow, tnt allow, wither allow, etc.)

What's wrong with it? Was that flags and settings created not for my target?

**Comment by wizjany** [comment]
your issue is that your first point and your 3rd through 5th points are contradictions. that's all there is to it. we don't support whatever use case you're trying to achieve.

**Comment by LaoTheLizard** [comment]
> your issue is that your first point and your 3rd through 5th points are contradictions. that's all there is to it. we don't support whatever use case you're trying to achieve.

But it worked so when I used WorldGuard on Minecraft 1.7.10. Why did you change this all? And why did you add that flags while them doesn't work?

**Comment by LaoTheLizard** [comment]
You know, I set up two servers on 1.7.10. And I set up config.yml for two different worlds (Overworld and Nether, for example) differently. In overworld wither couldn't break blocks. And in nether wither could. All worked excellent. What was wrong with it for you?

**Comment by LaoTheLizard** [comment]
My very thought was earlier: why did you add that flags? so many coder's moves for what? if they doesn't work.

**Comment by LaoTheLizard** [comment]
> your issue is that your first point and your 3rd through 5th points are contradictions. that's all there is to it. we don't support whatever use case you're trying to achieve.

https://prnt.sc/vphz9h
See that. I exploded TNT on the edge of two regions. One of them (right side) is mine, and another (left side) is not. So by default my TNT will explode just right side and left side will not have any damage. But at this picture you see the situation where I set flag tnt allow  to region __global__.  Both regions damaged by TNT. Is it wrong for you, right?
